src/use_case/Adopt.java

```diff
@@ -26,6 +26,7 @@ public void execute(AdoptInputData adoptInputData) {
         uwu.markUnavailable();
         List<String> users = userDAO.removePetFromAllUserBookmarks(uwu.getPetID());
         petDAO.save(uwu);
```

Comment:
u would need to loop thru users and do .addnotification("msg") on all of themi think